src/pages/lib/storage/move/q/platform/[platform].mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Move files`,
+  description: `Learn more about how to move files using Amplify's storage category.`
```

Comment:
for all of these pages where the feature is only available in the web and desktop dev preview, I think we need to be doing something more clear at the top of the page that this feature is ONLy available in dev preview, and link to where a developer can setup their dependencies to use that version.